pages/api/manifest.ts

```diff
@@ -8,29 +8,24 @@ import * as GeneratedGraphQL from "../../generated/graphql";
 import { getBaseURL } from "../../lib/middlewares";
 
 const capitalize = (value: string) => value.charAt(0).toUpperCase() + value.slice(1);
+const dropFileExtension = (filename: string) => path.parse(filename).name;
 
 const inferWebhooks = async (baseURL: string) => {
   const entries = await fg(['*.ts'], { cwd: 'pages/api/webhooks' });
 
-  const dropFileExtension = (filename: string) => path.parse(filename).name;
-  const r = entries.map(dropFileExtension).map((name: string) => {
-
+  return entries.map(dropFileExtension).map((name: string) => {
     const camelcaseName = name.split('-').map(capitalize).join('');
     const statement = `${camelcaseName}SubscriptionDocument`;
-    const query = statement in  GeneratedGraphQL ? print((GeneratedGraphQL as any)[statement]) : '';
+    const query = statement in  GeneratedGraphQL ? print((GeneratedGraphQL as any)[statement]) : null;
```

Comment:
one more comment here @zaiste , it was decided that manifest-based webhooks will require query to be in place, so we should figure out how to gracefully fail, when there's no subscription query in place, but I don't exactly know how we want to do so?

Reply:
I mean, with current setup this issue will appear only when requesting manifest, I believe it should be also failing when running `pnpm dev` or `pnpm build`.

Reply:
this API is still available as a Feature Preview so we don't need to decide right away. It'd be great that `query` equals `null` means a default, **full** payload. This should be implemented in the core. cc @patrys @timuric

Reply:
So I don't recall who made this decision, that `query` here will be required. If we will make it optional here, we could handle `syncEvents` here also (there was a discussion yesterday, also with @patrys  and yesterday was decided to just handle `asyncEvents` in manifest for the time being).

> It'd be great that query equals null means a default, full payload

I don't know, what does it mean full payload in this context, but before introduction of subscriptions we have (I called them at least as) legacy payloads and if query is not defined, that legacy payload is delivered.
The other question here is default payload for newly introduced eventtypes.